deploy/Chart/crds/radius/radapp.io_recipes.yaml

```diff
@@ -3,7 +3,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.15.0
+    controller-gen.kubebuilder.io/version: v0.9.1
```

Comment:
Not sure why updating go version to 1.22.0 downgraded this one to 0.9.1 @rynowak

Reply:
Maybe a bad merge/rebase? @youngbupark just updated this. Make sure you install the right controller-gen package and run `make generate`.

Reply:
Now it is using the right version.